cmd/catalog/start.go

```diff
@@ -66,6 +67,7 @@ func newRootCmd() *cobra.Command {
 	cmd.Flags().StringVar(&o.opmImage, "opmImage", defaultOPMImage, "the image to use for unpacking bundle content with opm")
 	cmd.Flags().StringVar(&o.utilImage, "util-image", defaultUtilImage, "an image containing custom olm utilities")
 	cmd.Flags().StringVar(&o.writeStatusName, "writeStatusName", defaultOperatorName, "ClusterOperator name in which to write status, set to \"\" to disable.")
+	cmd.Flags().Int64Var(&o.workloadUserID, "workload-user-id", 1001, "The non-root user ID that registry pods will run as.")
```

Comment:
```suggestion
```


The idea of passing the USER ID will not work.
We do have not a way to know a fixed value number valid for OCP. 
Also, that would mean ensuring that the ServiceAccount used would have permissions to the anyuid SCC.

Therefore, we have two options:
a) Ensure that the  USERID is define in the dockerfile image
OR 
b) Do not set ANY userID and do NOT set RunAsNonRoot 
Then, for downstream we need to ensure that the SA has access to the required SCC ( i.e. previleged )
Also, ensure that the namespace will be labeled


The valid range is based on the namespace config. 
Also, we cannot use RunAsUser. 

From OCP:

> 

If your S2I image does not include a USER declaration with a numeric user, your builds will fail by default. In order to allow images that use either named users or the root (0) user to build in OpenShift Container Platform, you can add the project’s builder service account (system:serviceaccount:<your-project>:builder) to the privileged security context constraint (SCC). Alternatively, you can allow all images to run as any user.
--

Reply:
Again, this conversation is valid for downstream, so we'll discuss this in the downstream PR.